lib/client.js

```
@@ -1,149 +1,160 @@
+        case 'connecting':
+          console.log('Connecting...');
+          break;
+
+        case 'close':
+          lastDisconnect.error?.output?.statusCode === DisconnectReason.loggedOut
+            ? manageProcess()
+            : client();
+          break;
+
+        case 'open':
+          const web = version.join('.');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)